.nyc_output/1ae8eccb94781ab868ab868a50dec857.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
`.nyc_output` folder seems like unused, right?

Reply:
I guess this .nyc_output was added by accident from an vscode extension. I'll remove that stuff again